filebeat/docs/fields.asciidoc

```diff
@@ -3896,7 +3896,7 @@ Index id
 *`elasticsearch.shard.id`*::
 +
 --
-type: keyword
+type: integer
```

Comment:
I think shard id is ok as `keyword`, even if it is always a number I guess it is only used as a keyword.

Reply:
@jguay raised the same the other day. I changed to keyword